og_ui/og_ui.info.yml

```diff
@@ -0,0 +1,11 @@
+name: Organic groups UI
```

Comment:
Oh it's in my nature to write "Organic Groups" with both words capitalized, but I saw it consistently spelled "Organic groups" in the code base, so I took care to use this form.